### PR TITLE
[OC-4912] [MCKIN-7492] Removed the themed-xblock selector to allow overrides by themes (v3)

### PR DESCRIPTION
**Pull request by xitij2000**
Fix issues with theme CSS overrides due to overly-specific selectors.

### CODE REVIEW

File: problem_builder/mentoring.py [hunk 1]
```diff
@@ -979,7 +977,6 @@ def student_view(self, context):
             'children_contents': children_contents,
         }))
         fragment.add_css_url(self.runtime.local_resource_url(self, 'public/css/problem-builder.css'))
-        fragment.add_css_url(self.runtime.local_resource_url(self, 'public/css/lms.css'))
```
Comment (pomegranited) [inline problem_builder/mentoring.py]: If we're removing this `lms.css` CSS file, then why bother keeping it (and editing it) below?

File: problem_builder/public/css/problem-builder.css [hunk 2]
```diff
@@ -118,16 +118,62 @@
 
 .mentoring .checkmark-incorrect {
     font-size: 22pt;
-    color: #ff0000;
+    color: #c1373f;
```
Comment (pomegranited) [inline problem_builder/public/css/problem-builder.css]: nit: where does this color come from?  Does it match an edX standard color somewhere?

File: problem_builder/mentoring.py (hunk 1 above)
Comment (xitij2000) [reply problem_builder/mentoring.py]: Oh sorry! I cherry-picked the v2 changes here, but I guess that doesn't delete files. I'll delete lms.css

File: problem_builder/public/css/problem-builder.css (hunk 2 above)
Comment (xitij2000) [reply problem_builder/public/css/problem-builder.css]: I just moved it from lms.css here. I don't see it used in edX, so not sure why this was picked.

Comment (pomegranited) [reply problem_builder/public/css/problem-builder.css]: Ah ok, understood.  👍

**Comment by xitij2000** [comment]
@pomegranited I updated the circle CI config to run ``apt-get update`` before installing packages, and it's fixed now. I'm keeping it a separate commit since it's not related to the rest of the work.

**Pull request merged**